discovery-provider/src/monitors/monitoring_queue.py

```diff
@@ -0,0 +1,75 @@
+import logging
+import time
+from src.monitors.monitors import MONITORS, get_monitor_redis_key
+from src.tasks.celery_app import celery
+
+logger = logging.getLogger(__name__)
+
+
+def refresh(redis, db, monitor):
+    """
+    Refreshes the cached value for a monitor
+
+    Args:
+        redis: Singleton redis instance
+        db: Singleton database instance
+        monitor: dict The monitor dictionary qwith name, func, ttl, and type
+    """
+    key = get_monitor_redis_key(monitor)
+    ttl_key = f"{key}:ttl"
+
+    is_fresh = redis.get(ttl_key)
+    if is_fresh:
+        return
+
+    value = monitor['func'](db=db, redis=redis)
```

Comment:
i think it would be better to use string consts instead of redefining 'func' in every usage

Reply:
same for the string usage in this PR generally

Reply:
I switched all of the consts to follow a similar convention to response_name_constants. hopefully a little less error prone

Reply:
ehhhhh not a big deal, i saw the later usage and it looks fine to me